text/0000-update_asset_packaging.md

```diff
@@ -38,66 +41,69 @@ Some reasons to stay with the current solution:
+
+_Cons:_
+* Each task needs manual set up and configuration.
+* Creates multiple intermediate files while running.
+
+__Option 2: Use Gulp task runner__
+
+_Pros:_
+* Runs faster then Grunt and with less configuration.
+* Tasks can be built as streams of smaller units.
+
+_Cons:_
+* Each task needs manual set up and configuration.
+* Little relevant knowledge within the current core team.
 
-__Option 2: Webpack + npm__
+__Option 3: Webpack__
 
 [POC PR](https://github.com/theforeman/foreman/pull/3433)
 
 _Pros:_
 * Live reload development server.
 * Ability to package the code in separate chunks that are only loaded when needed.
-* Multiple loaders allow for server-side tasks such as transpiling, linting, minifing etc. to be run without requiring an additional task runner.
+* Multiple loaders allow multiple tasks be run without requiring an additional task runner and with minimal configuration.
 * Can possibly replace rails asset pipeline in the future if desired.
-* npm uses CommonJS modules which is allows namespacing JS code, which means you just won't accidentally mess with your dependencies.
 
 _Cons:_
-* Complex configuration compared to other alternatives.
-* Lack of examples in other projects
+* More complex configuration compared to webpack.
+* Less examples in other projects.
+* Possible that some tasks may still require an additional runner.
 
-__Option 3: Browserify + npm__
+__Option 4: Browserify__
 
 [POC PR](https://github.com/theforeman/foreman/pull/3326)
 
 _Pros:_
 * Simple installation and configuration.
 * Easy to tie into rails asset pipeline.
-* Easy to add es2015, minification, etc.
-* npm uses CommonJS modules which is allows namespacing JS code, which means you just won't accidentally mess with your dependencies.
+* Possible to handle certain tasks with transforms.
 
 _Cons:_
-* Only handles JS assets.
+* Only handles JS assets in its core (some transforms may be able to help).
+* Possible that some tasks may still require an additional runner.
+* Integration with Bower seems more complex.
 
-__Option 4: Bower__
-
-[POC is part of PR](https://github.com/theforeman/foreman/pull/2882)
-
-_Pros:_
-* Bower seems to have a larger ecosystem for front-end assets then npm
-* Already in use in Katello, amount of effort required is halved.
-* Live reload development server could be done with some more work to grunt/gulp or any other build system
-* Bower uses flat dependency tree to reduce dependency duplicities that clients would have to download if plain npm + browserify was used.
-
-_Cons:_
-* Adds two more package managers instead of one - requires npm to install bower. This means we will have 3 different places that contain dependencies.
-* Some packages now are packaged only in npm
-* Does not provide solution for any server-side JS (testing, transpiling, etc.). Any such dependencies will need to be handled either by npm or bundler.
-
-__Option 5: npm alone__
-
-No POC available.
```

Comment:
If I understand the RFC process correctly then we aren't supposed to squash commits precisely for that reason.